lib/commands/unpublish.js

```diff
@@ -109,13 +109,17 @@ class Unpublish extends BaseCommand {
       const { content } = await pkgJson.prepare(localPrefix)
       manifest = content
     } catch (err) {
-      // we needed the manifest to figure out the package to unpublish
-      if (!spec) {
-        if (err.code === 'ENOENT' || err.code === 'ENOTDIR') {
+      if (err.code === 'ENOENT' || err.code === 'ENOTDIR') {
+        if (!spec) {
```

Comment:
Do we want some silly-level logging that we checked for a local package.json, and none was found? Might be useful if someone reports they provided a spec, and the publishConfig wasn't respected.

Reply:
The usage error stack will point right here.

Reply:
I was thinking primarily of the case where a spec is provided.